test/instrumentation.ts

```diff
@@ -0,0 +1,4 @@
+import { afterAll } from '@jest/globals';
```

Comment:
you need to import this file in tests to have it executed.

Reply:
I have implemented the globals now in both test files as I couldn't get it with `// eslint-disable-next-line @typescript-eslint/no-unused-vars` on the imports working.